hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/dispatcher/TransactionDispatcher.java

```diff
@@ -110,12 +116,43 @@ public Fees dispatchComputeFees(@NonNull final FeeContext feeContext) {
 
         try {
             final var handler = getHandler(feeContext.body());
+            if (shouldUseSimpleFees(feeContext)) {
+                var feeResult = handler.calculateFeeResult(feeContext);
+                return feeResultToFees(feeResult, fromPbj(feeContext.activeRate()));
+            }
             return handler.calculateFees(feeContext);
         } catch (UnsupportedOperationException ex) {
             throw new HandleException(ResponseCodeEnum.INVALID_TRANSACTION_BODY);
         }
     }
 
+    private boolean shouldUseSimpleFees(FeeContext feeContext) {
+        if (!feeContext.configuration().getConfigData(FeesConfig.class).simpleFeesEnabled()) {
+            return false;
```

Comment:
you could return `feeContext.configuration().getConfigData(FeesConfig.class).simpleFeesEnabled()`